src/editor/components/components/SceneEditTitle/SceneEditTitle.component.jsx

```diff
@@ -60,7 +60,7 @@ const SceneEditTitle = ({ sceneData }) => {
           <input
             className={styles.title}
             value={title}
-            onChange={handleChange}
+            // onChange={handleChange}
```

Comment:
remove?

Reply:
Judging by how I understood the code, there is no need to use the `onChange` handler for this `input`, at least now. Because the new scene title is entered in the prompt window. Related issue: https://github.com/3DStreet/3dstreet-editor/issues/311